examples/pytorch/ogb_lsc/MAG240M/experimental/train_unified_tensor.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import ogb
+from ogb.lsc import MAG240MDataset, MAG240MEvaluator
+import dgl
+import torch
+import numpy as np
+import time
+import tqdm
+import dgl.function as fn
+import numpy as np
+import dgl.nn as dglnn
+import torch.nn as nn
+import torch.nn.functional as F
+import argparse
+
+class RGAT(nn.Module):
```

Comment:
Can we just use from .train_multi_gpus import RGAT?

Reply:
Now the example code is changed to the GraphSAGE.